src/components/Calculator.js

```diff
@@ -0,0 +1,26 @@
+const Calculator = () => (
+  <section className="main">
+    <div className="display">0</div>
+    <div className="box">AC</div>
+    <div className="box">+/-</div>
+    <div className="box">%</div>
+    <div className="orange">÷</div>
+    <div className="box">7</div>
+    <div className="box">8</div>
+    <div className="box">9</div>
+    <div className="orange">x</div>
+    <div className="box">4</div>
+    <div className="box">5</div>
+    <div className="box">6</div>
+    <div className="orange">-</div>
+    <div className="box">1</div>
+    <div className="box">2</div>
+    <div className="box">3</div>
+    <div className="orange">+</div>
+    <div className="box zero">0</div>
+    <div className="box">.</div>
+    <div className="orange">=</div>
+  </section>
+);
+
```

Comment:
- [ ] Excellent work so far, 👏🏻 but according to the project requirement, the Calculator.js` component should be a class-based component and not a stateless function component. Kindly change this. 😄   💯 

- It should be like this:

```
class Calculator extends Component {
  render() {
    // reference as a js object
    return {
      <div>
       // Calculator app design goes here
      </div>
    };
  }
}

export default Calculator;